### DISCUSSION TITLE
New Filter UI: Where are all the Interfaces?

### DISCUSSION THREAD

**Discussion by shadowcpy** [directus/directus#8769] [Feature Requests]
Hi everyone,
I just recently tried out the new Filter UI in RC 96, but I am missing many of the Interfaces to select from when filtering fields...
In the old layout, a field filter would have the same interface for filtering like for the Item Detail page, but now it seems to be only implemented for a few select field types (Date, DateTime etc.) with all the others, like Multi-Select or Tree view being missing.

Is this intended or are the other interfaces planned? This severely degrades the workflow for normal users that do not want to enter everything manually.

**Comment by benhaynes** [comment]
As far as I remember, the only thing currently missing compared to the previous filter experience is the injection of "choices" (for things like dropdowns, multi-select, etc). These choices will be added back, yes.

**Comment by br41nslug** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
